main.js

```diff
@@ -118,24 +119,60 @@ function HexToRGB(hex) {
 };
 
 
-function oh2iob(type, val) {
+// type     : the type received from OH
+// val      : value
+// destType : destination type (needed eg. for OH color type)
+function oh2iob(type, val, destType = null, oldValue = null) {
```

Comment:
With destType = null you force Node.js >= 6.
Is it really required?

Reply:
No,  should work without it. I have added all parameters in the code. 
I am running node 8 and was not aware about this change. 
Sorry.